demo/grow-height-animation.html

```diff
@@ -9,7 +9,6 @@
 -->
 
 <link rel="import" href="../../polymer/polymer.html">
-<link rel="import" href="../../neon-animation/neon-animation-behavior.html">
```

Comment:
This import is for a behavior, I'm surprised it works without it - does the demo work, as in does this element animate?

Reply:
ah that's a good catch. I just saw `neon-animation` and all i saw was red. I'll add it back in